tests/stats/test_main.py

```diff
@@ -0,0 +1,132 @@
+import os
+import pytest
+from urllib import parse
+from zipfile import ZipFile
+from argparse import Namespace
+
+try:
+    import omegaconf  # noqa: F401
+    from dff.stats.__main__ import main
+    from dff.stats.cli import DEFAULT_SUPERSET_URL, DASHBOARD_SLUG
+    from dff.stats.utils import get_superset_session
+except ImportError:
+    pytest.skip(reason="`OmegaConf` dependency missing.", allow_module_level=True)
+
+from tests.db_list import SUPERSET_ACTIVE
+from tests.test_utils import get_path_from_tests_to_current_dir
+
+path_to_addon = get_path_from_tests_to_current_dir(__file__)
+
+
+def dashboard_display_test(args: Namespace, base_url: str):
+    dashboard_url = parse.urljoin(base_url, f"/api/v1/dashboard/{DASHBOARD_SLUG}")
+    charts_url = parse.urljoin(base_url, "/api/v1/chart")
+    datasets_url = parse.urljoin(base_url, "/api/v1/dataset")
+    database_conn_url = parse.urljoin(base_url, "/api/v1/database/test_connection")
+    sqla_url = f"clickhousedb+connect://{getattr(args, 'db.user')}:{getattr(args, 'db.password')}@clickhouse:8123/test"
```

Comment:
Why not get all attributes (port, db name, e.t.c.) from `args`?

Reply:
Done